### DISCUSSION TITLE
Code Generation for apollo-link-rest

### DISCUSSION THREAD

**Discussion by wootencl** [dotansimha/graphql-code-generator#8448] [Ideas] [answered]
Just wondering if there's any movement in the direction to support apollo's `apollo-link-rest` which supports using the apollo client with with RESTful APIs. Certainly understand this codegen is meant for graphql APIs but with `apollo-link-rest`'s syntax it seems there may be some potential to support codegen in that direction. 

Thanks!

**Comment by charlypoly** [comment, marked as answer]
Hi @wootencl,

We don't support and plan to write a new plugin for `apollo-link-rest`.
However, we welcome all new contributions to the codegen plugins hub!

https://www.the-guild.dev/graphql/codegen/docs/custom-codegen
